packages/terra-functional-testing/src/services/wdio-visual-regression-service/utils/strategies/BaseStrategy.js

```diff
@@ -13,25 +13,18 @@ class BaseStrategy {
   }
 
   setScrollArea(startX, startY, endX, endY) {
-    // TODO: bug https://github.com/zinserjan/wdio-screenshot/issues/87
-    // const documentWidth = this.screenDimensions.getDocumentWidth();
-    // const documentHeight = this.screenDimensions.getDocumentHeight();
-
-    // if (startX >= documentWidth) {
-    //   throw new Error('startX is out of range');
-    // } else if (startY >= documentHeight) {
-    //   throw new Error('startY is out of range');
-    // } else if (endX > documentWidth) {
-    //   throw new Error('endX is out of range');
-    // } else if (endY > documentHeight) {
-    //   throw new Error('endY is out of range');
-    // }
+    const documentWidth = this.screenDimensions.getDocumentWidth();
+    const documentHeight = this.screenDimensions.getDocumentHeight();
+    const adjustedStartX = startX > documentWidth ? documentWidth : startX;
```

Comment:
The adjustedStart X & Y seem a bit weird. If I'm following this correctly, this is saying the start x is off the screen? Thinking of an example being a closed slide panel, would the resulting image be a 0px by 0px if both start and end 7 values are the doc width?

Reply:
Yes, in the case where startX and StartY are > documentWidth and documentHeight respectively, this means the selector element is 100% off the screen and it would be a 0px image. Would it make more sense to default adjustedStartX/adjustedStartY to 0,0 so that it would the full screen?

```
    const adjustedStartX = startX > documentWidth ? 0 : startX;
    const adjustedStartY = startY > documentHeight ? 0 : startY;
```